Fix Mustache Lint fail

Ci Test Result with the following error :
Run Mustache Lint on format_twocol
/home/runner/work/moodle-format_twocol/moodle-format_twocol/moodle/course/format/twocol/templates/course_summary.mustache - WARNING: HTML Validation error, line 15: CSS: “background-color”: Parse Error. (          <div class="course-img course-img-"                                style='background-color: ;'            >     )
/home/runner/work/moodle-format_twocol/moodle-format_twocol/moodle/course/format/twocol/templates/course_summary.mustache - WARNING: HTML Validation info, line 31: Empty heading. (	        <h2></h2>)
/home/runner/work/moodle-format_twocol/moodle-format_twocol/moodle/course/format/twocol/templates/course_summary.mustache - WARNING: HTML Validation info, line 43: Empty heading. (          <h3></h3>)
/home/runner/work/moodle-format_twocol/moodle-format_twocol/moodle/course/format/twocol/templates/course_summary.mustache - WARNING: HTML Validation info, line 50: Empty heading. (          <h3></h3>)
/home/runner/work/moodle-format_twocol/moodle-format_twocol/moodle/course/format/twocol/templates/course_summary.mustache - WARNING: HTML Validation error, line 52: Bad value “region” for attribute “role” on element “li”. (          <li id="section-0" class="section main clearfix" role="region" aria-label="overview">     )
/home/runner/work/moodle-format_twocol/moodle-format_twocol/moodle/course/format/twocol/templates/local/content.mustache - INFO: Template produced no content
/home/runner/work/moodle-format_twocol/moodle-format_twocol/moodle/course/format/twocol/templates/local/content.mustache - OK: Mustache rendered html succesfully
/home/runner/work/moodle-format_twocol/moodle-format_twocol/moodle/course/format/twocol/templates/course_topic.mustache - WARNING: HTML Validation error, line 7: CSS: “background-color”: Parse Error. (>        <div class="section-navigation navigationtitle course-img course-img-"                        style='background-color: ;'        >     )
/home/runner/work/moodle-format_twocol/moodle-format_twocol/moodle/course/format/twocol/templates/course_topic.mustache - WARNING: HTML Validation info, line 24: Empty heading. (          <h2 class=""></h2>)
/home/runner/work/moodle-format_twocol/moodle-format_twocol/moodle/course/format/twocol/templates/progress_bar.mustache - OK: Mustache rendered html succesfully
/home/runner/work/moodle-format_twocol/moodle-format_twocol/moodle/course/format/twocol/templates/course_summary_reverse.mustache - WARNING: HTML Validation info, line 13: Empty heading. (          <h3></h3>)
/home/runner/work/moodle-format_twocol/moodle-format_twocol/moodle/course/format/twocol/templates/course_summary_reverse.mustache - WARNING: HTML Validation info, line 20: Empty heading. (          <h3></h3>)
/home/runner/work/moodle-format_twocol/moodle-format_twocol/moodle/course/format/twocol/templates/course_summary_reverse.mustache - WARNING: HTML Validation error, line 22: Bad value “region” for attribute “role” on element “li”. (          <li id="section-0" class="section main clearfix" role="region" aria-label="overview">     )
/home/runner/work/moodle-format_twocol/moodle-format_twocol/moodle/course/format/twocol/templates/course_summary_reverse.mustache - WARNING: HTML Validation error, line 36: CSS: “background-color”: Parse Error. (          <div class="course-img course-img-"                                style='background-color: ;'            >     )
/home/runner/work/moodle-format_twocol/moodle-format_twocol/moodle/course/format/twocol/templates/course_summary_reverse.mustache - WARNING: HTML Validation info, line 52: Empty heading. (	        <h2></h2>)
Error: Process completed with exit code 1.

--- a/templates/course_summary.mustache
+++ b/templates/course_summary.mustache
@@ -21,7 +21,10 @@
 
     Example context (json):
     {
-
+        "headerbackcolor": "#FFFFFF",
+        "summaryname": "Summary",
+        "detailsheading": "Details",
+        "resourcesheading": "Resources"
     }
 }}
 
@@ -127,7 +130,7 @@
 
           <h3>{{resourcesheading}}</h3>
           <ul class="twocol">
-           <li id="section-0" class="section main clearfix" role="region" aria-label="overview">
+           <li id="section-0" class="section main clearfix" aria-label="overview">
                 <span class="hidden sectionname">overview</span>
                 <div class = "content">
                     {{#mods}}

--- a/templates/course_summary_reverse.mustache
+++ b/templates/course_summary_reverse.mustache
@@ -21,7 +21,10 @@
 
     Example context (json):
     {
-
+        "headerbackcolor": "#FFFFFF",
+        "summaryname": "Summary",
+        "detailsheading": "Details",
+        "resourcesheading": "Resources"
     }
 }}
 
@@ -86,7 +89,7 @@
 
           <h3>{{resourcesheading}}</h3>
           <ul class="twocol">
-           <li id="section-0" class="section main clearfix" role="region" aria-label="overview">
+           <li id="section-0" class="section main clearfix" aria-label="overview">
                 <span class="hidden sectionname">overview</span>
                 <div class = "content">
                     {{#mods}}

--- a/templates/course_topic.mustache
+++ b/templates/course_topic.mustache
@@ -21,7 +21,8 @@
 
     Example context (json):
     {
-
+        "headerbackcolor": "#FFFFFF",
+        "sectionname": "Topic"
     }
 }}
 <div class="twocol">

